hazelcast/config.py

```diff
@@ -435,6 +438,21 @@ def cluster_name(self, value: str) -> None:
 
         self._cluster_name = value
 
+    @property
```

Comment:
Just a note, we should not add a configuration for this task. You should use the port you selected as the port of the addresses in the `cluster_members` config option

Reply:
I did not get what do you mean

Reply:
We use the port from the cluster member addresses and should put that into there. 
```python
client = HazelcastClient(cluster_members=["localhost:12345"])
```
when you do your changes, you need to put the port you selected above, not into a new config you created here.

Reply:
Okay, reverting commit then.